components/raftstore/src/coprocessor/region_info_accessor.rs

```diff
@@ -1166,8 +1166,8 @@ mod tests {
         ];
 
         for index in indices {
-            for order in orders {
-                test_split_impl(*index, *order);
+            for &order in orders {
```

Comment:
`*order` is a prefered style than `&order` matching.

Reply:
I prefer this than putting a `[allow(explicit_auto_deref)]`.

Reply:
Then it's a false alarm from clippy as "deref which would be done by auto-deref" is false in this case. You can get around the problem by `order.as_slice()` or give orders a explicit type `orders: &[&[usize]]`.